File: OrmAppWithAsp/OrmAppWithAsp/Model/Student.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace OrmAppWithAsp.Model
{
    public class Student
    {
        [Key]
        public long Id { get; set; }

        public string FullName { get; set; }

        [Column("Start")]
        public DateTime DateStart { get; set; }

        public string Email { get; set; }

        public string Phone { get; set; }

        public double Note { get; set; }
    }
}

File: orm_mysql/src/main/java/com/example/orm_mysql/model/Student.java
package com.example.orm_mysql.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Entity
@Data @AllArgsConstructor @NoArgsConstructor @Builder
public class Student {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String fullName;
    private Date dateStart;
    private String email;
    private String phone;
    private double note;
}
